client/src/pages/Home.css

```diff
@@ -9,3 +9,46 @@
 .message:hover {
 	background-color: yellow;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(214, 227, 243);
+  align-items: center;
+  height: 100%;
+  border-radius: 12px;
+  padding: 100px 100px 100px 100px;
+  width: 70%;
+  margin-left: 10em;
```

Comment:
We don't need to use a large margin here because we'll have too large a margin on mobile. Instead, we can add the `container` class from bootstrap to the `main` element in `Home.js` so that there is space around the edges of the UI